.env.example

```
@@ -0,0 +1,2 @@
+.env.example 
```

dotenv-linter (4.0.0)

[warning] [IncorrectDelimiter] The .env.example key has incorrect delimiter

(IncorrectDelimiter)